bee-snapshot/src/local/snapshot.rs

```diff
@@ -186,6 +187,11 @@ impl LocalSnapshot {
                 Ok(_) => u64::from_le_bytes(buf_value),
                 Err(e) => return Err(SnapshotReadError::IOError(e)),
             };
+
+            if i % 10_000 == 0 && i != 0 {
+                info!("Read {} balances.", i);
+            }
```

Comment:
This is nice, but if the intent is showing progress, I'd rather log a % than a number. A number doesn't really help if you don't show the upper bound as well.

Reply:
Sounds good. I'll change that. Also....instead of printing progress line by line I think I'ld prefer resetting a single line. I tried it out already and it looks way better. Originially I tried to re-create the Cargo compilation progress indicator:
Example:
[================>               ]  178/203
but simpler would be just:
Read balances: 24000/40000 (35%)
then rewriting this line instead of creating a new line each time. Wdyt?

Reply:
That would definitely be nicer but given that the other tasks are progressing as well you can't really reset the line.

Reply:
You're probably right. But currently we can because reading the snapshot file is done before the actual node operations start. In my tests on the live Bee nodes nothing ever interfered with this progress indicator.

Reply:
Hmm that's not true, you changed that in the generic tangle PR. Running a node node does lots of things before the LS parsing is finished.

Reply:
hm, okay. that's interesting. I am not sure but maybe I never saw an issue with the one-line progressbar because I developed this on a different branch which didn't have the generic tangle merged. :thinking:

Reply:
anyway, that was just for the fun of it anyway. I am fine with new-line progress indicator with percentages.

Reply:
Yeah that's fine for a start, we'll do better later